packages/sanity/src/core/form/inputs/DateInputs/DateTimeInput.tsx

```diff
@@ -37,19 +43,28 @@ function parseOptions(options: SchemaOptions = {}): ParsedOptions {
     dateFormat: options.dateFormat || DEFAULT_DATE_FORMAT,
     timeFormat: options.timeFormat || DEFAULT_TIME_FORMAT,
     timeStep: ('timeStep' in options && Number(options.timeStep)) || 1,
+    displayTimezone:
+      options.displayTimezone && isValidTimezoneString(options.displayTimezone)
+        ? options.displayTimezone
+        : undefined,
   }
 }
 
 function serialize(date: Date) {
   return date.toISOString()
 }
-function deserialize(isoString: string): ParseResult {
-  const deserialized = new Date(isoString)
-  if (isValidDate(deserialized)) {
-    return {isValid: true, date: deserialized}
+
+// deserialize specifically handles the deserialisation of the iso date string from content lake
+const deserialize =
```

Comment:
I feel like this should be named `getDeserializer` or similar - it read as a mistake to me when I later on read `deserialize(timezone)`